svcs/guacamole/guacamole.go

```diff
@@ -134,7 +154,8 @@ func (guac *guacamole) create(ctx context.Context) error {
 		},
 	})
 
-	mysqlPass := uuid.New().String()
+	mysqlPass := "deneme"
```

Comment:
I'm not really for hardcoding passwords

Reply:
I have used it while I was trying to understand how data stores. I will re-add randomised one of course.

Reply:
replaced.